examples.Rmd

```diff
@@ -193,3 +193,44 @@ profvis({
 ```
 
 This version of the code, with `csum2`, is around 60x faster than our original code, and requires almost no memory allocation. These performance improvements were possible by avoiding calls to `$` and `$<-`, and by avoiding unnecessary memory allocation and copying from growing a vector in a loop.
+
+
+### Example 3 - Profiling a Shiny Application
+
+In addition to R code, you can also profile [Shiny](shiny.rstudio.com) applications. To do this, simply execute the `runApp()` command inside of `profvis`. For instance, we can run one of shiny's default examples using the `runExample` command (which is a wrapper for `runApp`).
+
+```{r,eval=FALSE, echo=FALSE}
+#This code block is what we run, it must be manually run each time
+library(shiny)
+p<-profvis({
+  runExample(example = '06_tabsets', display.mode ="normal")
+})
+saveRDS(p, "shinyapp.rds")
+
+```
+
+
+```{r,eval=FALSE}
+library(shiny)
+profvis({
+  runExample(example = '06_tabsets', display.mode ="normal")
```

Comment:
Double quotes instead of single quotes, and space after `=` (and same in the code block above).